manifests/templates/otel-collector.yaml

```diff
@@ -93,7 +93,7 @@ spec:
     spec:
       containers:
       - name: otel-collector
-        image: gcr.io/config-management-release/otelcontribcol:v0.89.0-gke.1
+        image: gcr.io/config-management-release/otelcontribcol:v0.91.0-gke.0
```

Comment:
Looks like this may need a bump for `golang.org/x/crypto`

Reply:
/hold

Patched the version. I'll bump the image version later.

Reply:
updated